source/fundamentals/indexes.txt

```diff
@@ -242,9 +244,20 @@ language as an option when creating the index.
    Text indexes differ from the more powerful :atlas:`Atlas full text search indexes </atlas-search/>` Atlas users
    should use Atlas search.
 
-.. warning::
+.. warning:: Restriction on one text index per collection. 
+
+   If you want to create a text index for multiple text fields, you need
+   to create an compound index to include them. 
 
-   Collections are limited to one text index. Keep in mind that a text index can cover multiple fields.
+   The following creates a compound text index for the ``title`` and ``genre``
+   fields:
+
+   .. code-block:: java
+   
+      collection.createIndex(Indexes.compoundIndex(Indexes.text("title"), Indexes.text("genre")));
+   
+   For more information on compound text index restrictions, see the
```

Comment:
Nitpick: "compound text index" is a long description to precede "restrictions". Instead, consider "For more information on restrictions on compound text indexes, ..."

Reply:
Reformatted this to a "for more information, see the following:" part